articles/install-guide/toc.yml

```diff
@@ -5,9 +5,9 @@ items:
   href: standalone.md
 - name: Q# Jupyter notebooks
   href: qjupyter.md
-- name: Using Q# with other languages
+- name: Q# with other languages
```

Comment:
I think this should be consistent across the TOC and articles, if it's agreed that "Develop" is the right verb. 
Develop with Q# and <target> [Python, .NET, other languages, Jupyter Notebooks...]

Reply:
But won't get "Develop with Q# and other languages"  and "Develop with Q# Jupyter Notebooks" too long for the TOC? That's the main reason why I didn't add "Develop with..." to the TOC.

Reply:
Personally I'm for keeping "Develop with..." out in the TOC just because it bogs it down with clutter.